routes/api.js

```diff
@@ -7,6 +7,14 @@ var debug = require('debug')('dashboard-proxy:server');
 var error = require('debug')('dashboard-proxy:server:error');
 var Buffer = require('buffer').Buffer;
 var BufferList = require('../node_modules/websocket/vendor/FastBufferList');
+var BufferList;
+try {
+    // from source tree
+    BufferList = require('../node_modules/websocket/vendor/FastBufferList');
+} catch(e) {
+    // from install location
+    BufferList = require('../../websocket/vendor/FastBufferList');
+}
```

Comment:
Oh geez. That was a rebase problem. Let me fix that at least. But yes, let's find something cleaner if we can.